docs/guides/main.rst

```diff
@@ -139,12 +138,13 @@ place to start::
     directory: ~/music
     library: ~/data/musiclibrary.db
 
-Change that first path to a directory where you'd like to keep your music. Then,
-for ``library``, choose a good place to keep a database file that keeps an index
+This should work just fine for Windows users, the ``~`` is equivalent to the
+%USERPROFILE% - typically, ``C:\Users\Your_Machine_LoginName``). Change that 
```

Comment:
Good to clarify that this works for Windows users too, but I'm going to move this until after we've introduced the meaning of the two paths.